### PR TITLE
lightspeed: use the code value from the 403

### PR DESCRIPTION
**Pull request by goneri**
The 403 error can now return a `code` value to specify the nature of the
error.
`permission_denied_wca_api_key_is_missing` is the first case we can get.
The error happens when the access is denied because the user has a seat
but the organization has not been fully configured yet.

Depends-On: https://github.com/ansible/vscode-ansible/pull/1026
Depends-On: https://github.com/ansible/vscode-ansible/pull/1029

### CODE REVIEW

**Review by ganeshrn — APPROVED** [review]

**Comment by ganeshrn** [comment]
@goneri Can you please fix the CI failure

File: src/features/lightspeed/lightSpeedOAuthProvider.ts [hunk 1]
```diff
@@ -630,4 +636,24 @@ export class LightSpeedAuthenticationProvider
...
+      );
+      return undefined;
+    } else if (authSession?.rhOrgHasSubscription) {
+      console.log(
+        `[ansible-lightspeed-oauth] User "${authSession?.account?.label}" has an Org with a subscription.`
+      );
+      return true;
+    } else {
+      console.log(
+        `[ansible-lightspeed-oauth] User "${authSession?.account?.label}" does have an Org with a subscription.`
```
[the remark this answers is not in the document]
Comment (goneri) [reply src/features/lightspeed/lightSpeedOAuthProvider.ts]: I've fixed the message. It was indeed not correct. Thanks @ajinkyau.

**Review by ajinkyau — APPROVED** [review]

**Comment by goneri** [comment]
The CI gods are against me.

**Comment by goneri** [comment]
@ajinkyau could you also take a look at https://github.com/ansible/vscode-ansible/pull/1029. It's a CI fix that address the err 127 that we get time to time when Task tries to call `yarn`.

**Pull request merged**